install-dev/langs/en/data/tab.xml

```diff
@@ -133,4 +133,5 @@
     <tab id="Zones" name="Zones" />
     <tab id="Marketing_tools" name="Module Selections" />
     <tab id="Tabs" name="Menu" />
+    <tab id="FeatureFlag" name="Experimental Features" />
```

Comment:
And this one @matks  ?

Reply:
This one is expected. It's the name of the tab, displayed to merchant. We want it to be displayed "Experimental features" (and then translate it into different languages).

"Feature Flag" is only the technical name. Displayable elements like page name, tab name, listing name must output "Experimental Features" in English.

Reply:
:+1: